integrationtest/wildfly/src/wildflyConfig/standalone/configuration/standalone-full-testqueues.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
```

Comment:
I assume you took a fresh file from WF 10 and re-applied our previous customizations?